components/_patterns/01-atoms/02-text/00-headings/h1/h1.twig

```diff
@@ -0,0 +1,28 @@
+{#
+/**
+ * Available variables:
+ * - heading - the content of the heading (typically text)
+ * - heading_url - the url the heading should point to
+ * - h1_base_class - the base class name
+ * - modifiers - array of modifiers to add to the base classname
+ * - blockname - blockname prepended to the base classname
+ * - heading_link_base_class - override the link base class
+ * - heading_link_blockname - override the link block name
+ */
+#}
+{% set h1_base_class = h1_base_class|default('h1') %}
+
+<h1{{ bem(h1_base_class, (modifiers), blockname) }}>
```

Comment:
So, one thing I found out using this on Ithaca, is that if we just use 'modifiers' at all levels, it gets passed down the chain. So, for example, if we set `modifiers = [alt, light]` to get `class="h1-alt h1-light` they'll also get passed down to the link component because it also uses the variable `modifiers`. So we'll end up with something like this:
```
<h1 class="h1-alt h1-light">
  <a class="link-alt link-light></a>
</h1>
```
(I've intentionally left out the href etc. to focus on the classes)

I didn't want that in my project, so I used the following in my link and h1 files respectively
```
<a{{ bem(link_base_class, (link_modifiers), link_blockname) }}>
```
and
```
+<h1{{ bem(heading_base_class, (heading_modifiers), heading_blockname) }}>